### PR TITLE
Rewritting of the authentication process

### PR DESCRIPTION
**Pull request by Max-7**

### CODE REVIEW

**Review by alexandrediasldev — COMMENTED** [review]
Working as intended, just some minor adjustement to do

File: parsec/core/gui/tr/parsec_en.po [hunk 1]
```diff
@@ -1542,17 +1569,21 @@ msgstr ""
...
 msgid "ACTION_LOGGING_IN"
 msgstr "Logging in..."
 
 msgid "ACTION_LOG_IN"
 msgstr "Log In"
 
+msgid "TEXT_LOGIN_ENTER_PASSWORD_INSTRUCTIONS_organization-device-user-email"
+msgstr ""
+"Welcome back to PARSEC <b>{user}</b>! Please enter the password of your "
+"<b>{device}</b> device."
```
[the remark this answers is not in the document]
Comment (touilleMan) [reply parsec/core/gui/tr/parsec_en.po:1594]: done.

File: parsec/core/gui/tr/parsec_en.po [hunk 2]
```diff
@@ -1542,17 +1569,21 @@ msgstr ""
 "\n"
 "</body></html>"
 
-msgid "TEXT_LOGIN_ENTER_PASSWORD_INSTRUCTIONS_organization-device-user-email"
-msgstr ""
-"Welcome back to PARSEC <b>{user}</b>! Please enter the password of your "
-"<b>{device}</b> device."
+msgid "TEXT_LOGIN_SELECT_SMARTCARD_INSTRUCTIONS_organization-device-user-email"
+msgstr "Welcome back to PARSEC <b>{user}</b>! You will be asked to provide "
+"your smartcard to authenticate to <b>{device}</b>."
```
[the remark this answers is not in the document]
Comment (touilleMan) [reply parsec/core/gui/tr/parsec_en.po]: done.

File: parsec/core/gui/tr/parsec_fr.po [hunk 3]
```diff
@@ -2787,3 +2799,33 @@ msgstr "Trouver un fichier via son lien"
...
+#~ "veuillez entrer votre ancien mot de "
+#~ "passe :"
+
+#~ msgid "TEXT_LABEL_PASSWORD_OLD"
+#~ msgstr "Mot de passe actuel"
+
+#~ msgid "TEXT_PASSWORD_CHANGE_NEW_PASSWORD_INSTRUCTIONS"
+#~ msgstr "Puis choisissez-en un nouveau :"
+
+#~ msgid "ACTION_CHANGE_PASSWORD"
```
[the remark this answers is not in the document]
Comment (touilleMan) [reply parsec/core/gui/tr/parsec_fr.po]: done.

**Pull request merged**